test/op_shape_test.cpp

```diff
@@ -431,7 +431,7 @@ TEST_CASE(contiguous_dyn_shape)
 
 TEST_CASE(contiguous_shape_scalar)
 {
-    migraphx::shape output{migraphx::shape::float_type};
+    migraphx::shape output{migraphx::shape::float_type, {1}};
```

Comment:
Can we add a test inside `shape_test` for standard() where one of the dims is 1 and stride for for that dim is a random value? It should assert that it is a standard shape.

Reply:
Also test add test for `contiguous` in `op_shape_test` where it converts such standard shape to "standard"-standard shape.

Reply:
it seems like contiguous does not convert modify the strides for such a shape. Is that the expected behaviour?